zhaquirks/tuya/tuya_valve.py

```diff
@@ -666,3 +670,38 @@ class GiexIrrigationStatus(t.enum8):
     .skip_configuration()
     .add_to_registry()
 )
+
+
+# Tuya 214C Ultrasonic water meter valve
+(
+    TuyaQuirkBuilder("_TZE200_zlwr0raf", "TS0601")
+    .tuya_metering(dp_id=1, metering_cfg=TuyaValveWaterConsumed)
+    # Skipped DP 2,3,4,5,6,16,18
+    .tuya_onoff(dp_id=13)
+    .tuya_switch(
+        dp_id=14,
+        attribute_name="auto_clean",
+        entity_type=EntityType.CONFIG,
+        translation_key="auto_clean",
+        fallback_name="Auto clean",
+    )
+    .tuya_dp(
+        dp_id=21,
+        ep_attribute=TuyaValveWaterConsumed.ep_attribute,
+        attribute_name=Metering.AttributeDefs.instantaneous_demand.name,
+    )
+    .tuya_temperature(dp_id=22)
+    .tuya_sensor(
+        dp_id=26,
+        attribute_name="voltage",
+        type=t.uint16_t,
+        converter=lambda x: x * 100,
+        state_class=SensorStateClass.MEASUREMENT,
+        unit=UnitOfElectricPotential.VOLT,
+        entity_type=EntityType.STANDARD,
+        translation_key="voltage",
```

Comment:
We should use the voltage device class and no translation key. Similar to: https://github.com/zigpy/zha-device-handlers/blob/dd63a95fbbe99c2544ac4d56077f25bf86701cfd/zhaquirks/tuya/tuya_thermostat.py#L270-L278